fix: resourcePath in authorizers should contain wildcards

danilofuchs opened the pull request:
Fixes https://github.com/dherault/serverless-offline/issues/917

I still need to write tests for this fix.

I must say, the development experience to contribute to this project for the first time is quite poor. 
The main issue is I could not run tests reliably in my machine:
- every single time I want to run tests, npm installs every 'scenario' dependencies
- I cannot run unit tests separately, only the whole project with many integration tests
- I'm not sure where to add tests, considering unit tests are in a folder `old-unit`. I'm not confident in adding code to something with `old` in its name
- Some random tests break by timeout
- Husky runs tests on every push

In my opinion, tests should be easy to run locally, but not enforced to be ran on every push. Travis CI already does this tests for us, and is way more convenient. Running them locally on every push (which takes some minutes) will certainly make developers reconsider contributing ever again. For instance, [Next.js](https://github.com/zeit/next.js/blob/canary/contributing.md) is an immense project, and allows for flexible testing locally, and tests every PR on a CI.

If you would like, I can open an issue with my concerns so we can discuss it separately

dherault commented:
Hi @danilofuchs thanks for your PR!

"Some random tests break by timeout" --> I know, it sucks, did not write those. I tend to forget about them.
"Husky runs tests on every push" --> Not anymore, ouf.

chardos commented:
Hi @danilofuchs, thanks for putting in that PR!

Would you be able to resolve the conflicts before we merge?

danilofuchs commented:
> Would you be able to resolve the conflicts before we merge?

Sure! I just rebased my branch and added a simple integration test.

Thank you for taking my concerns into consideration. I now see it is much easier to test the repository
